scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/RunProcessor.java

```diff
@@ -87,7 +87,9 @@ public Platform getPlatformType() {
     Stream<Builder> standard =
         Stream.of(
             new Builder(Platform.ILLUMINA, "default", DefaultIllumina::create),
-            new Builder(Platform.PACBIO, "default", DefaultPacBio::create));
+            new Builder(Platform.PACBIO, "default", DefaultPacBio::create),
+            new Builder(Platform.OXFORDNANOPORE, "promethion", DefaultPromethion::create) /*,
+            new Builder(Platform.OXFORDNANOPORE, "minion", DefaultMinion::create)*/);
```

Comment:
remove commented-out code

Reply:
This code will be uncommented when the MinION processor is ready.

Reply:
We try to keep code clean of commented code and TODO notes as these make more sense in other branches and JIRA tickets respectively